script/state.py

```diff
@@ -0,0 +1,34 @@
+import json
+import psycopg2
+
+hostname = "postgres"
+database = "geovisualizer"
+username = "postgres"
+
+port_id = 5432
+conn = psycopg2.connect(host=hostname, dbname=database, user=username, port=port_id)
+cur = conn.cursor()
+
+cur.execute("SELECT id,state,country,ST_AsGeoJSON(boundary,4326)::JSONB from states")
+query = cur.fetchall()
```

Comment:
These are results of the query, hence can be named better like `query_results` or `states_data`